src/js/components/Button/__tests__/Button-test.js

```diff
@@ -116,41 +113,39 @@ describe('Button', () => {
   });
 
   test('primary', () => {
-    const component = renderer.create(
+    const { container } = render(
       <Grommet>
         <Button primary label="Test" onClick={() => {}} />
       </Grommet>,
     );
-    const tree = component.toJSON();
-    expect(tree).toMatchSnapshot();
+
+    expect(container.firstChild).toMatchSnapshot();
   });
 
   test('color', () => {
-    const component = renderer.create(
+    const { container } = render(
       <Grommet>
         <Button color="accent-1" label="Test" onClick={() => {}} />
         <Button color="accent-1" primary label="Test" onClick={() => {}} />
         <Button color="#111111" primary label="Test" onClick={() => {}} />
         <Button color="#123" primary label="Test" onClick={() => {}} />
       </Grommet>,
     );
-    const tree = component.toJSON();
-    expect(tree).toMatchSnapshot();
+
+    expect(container.firstChild).toMatchSnapshot();
   });
 
   test('fill', () => {
-    const component = renderer.create(
+    const { container } = render(
       <Grommet>
-        <Button>
-          <Button fill />
-          <Button fill={false} />
-          <Button fill="horizontal" />
-          <Button fill="vertical" />
-        </Button>
```

Comment:
Good catch here, I agree that it doesn't make sense for the 4 Buttons to be wrapped within a Button for this test. I think this is a good change that simplifies the test.

Reply:
Thanks!

I've noticed there are some other Button tests we can improve but I tried not to do it in the same PR to not make it hard to understand.

Reply:
Yeah good call not doing those other changes in this PR, but if you think some of the other tests could be improved definitely feel free to file a separate PR with those changes

Reply:
Cool, I'll do it soon